tests/test_all_rules.py

```diff
@@ -632,7 +632,8 @@ def test_integration_tag(self):
 
                     # checks if the rule path matches the intended integration
                     # excludes BBR rules
-                    if rule_integration in valid_integration_folders:
+                    if rule_integration in valid_integration_folders and \
+                            not hasattr(rule.contents.data, 'building_block_type'):
                         if rule.path.parent.name not in rule_integrations and rule.path.parent.name != "bbr":
```

Comment:
```suggestion
                        if rule.path.parent.name not in rule_integrations:
```
Do we still need the ` rule.path.parent.name != "bbr":` if we are checking for `not hasattr(rule.contents.data, 'building_block_type')` in the prior if statement?